lms/resources/lti_launch/base.py

```diff
@@ -137,17 +115,7 @@ def ext_lti_assignment_id(self):
         return self._request.POST.get("ext_lti_assignment_id")
 
     @property
-    def is_canvas(self):
-        """Return True if Canvas is the LMS that launched us."""
-        if (
-            self._request.parsed_params.get("tool_consumer_info_product_family_code")
-            == "canvas"
-        ):
-            return True
-
-        if "custom_canvas_course_id" in self._request.parsed_params:
-            return True
-
+    def is_canvas(self) -> bool:
```

Comment:
Again, if we continue with this pattern I don't see a reason to do `is_canvas` very often if we use a common interface.